insights/parsers/mongod_conf.py

```diff
@@ -80,14 +80,15 @@ class MongodbConf(Parser, LegacyItemAccess):
     """
     Parse config file in key-value pair or YAML format.
     Make several frequently used config options as properties.
+    Attribute 'is_yaml' is a boolean value, flags if this is a yaml format file.
```

Comment:
You might want to update the docsting of the attribute of `is_yaml` to list it together with below properties.

You can check it in your local by making the html format document:
~~~
cd docs; make html
google-chrome-stable docs/_build/html/shared_parsers_catalog/mongod_conf.html
~~~